styles.css

```diff
@@ -1,3 +1,345 @@
+}
+
+.intro h4 {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  margin-top: 30px;
+  margin-bottom: 0;
+}
+
+.intro p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.intro button {
+  padding: 12px;
+  background: #ff6b00;
+  border: 0 solid transparent;
+  color: white;
+}
+
+.langs h3 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #fff;
+  margin: 20px 0;
+}
+
+.bg-about {
+  background-image: url('./Assets/Groupx.png');
+  background-repeat: no-repeat;
+  background-position: right -0 bottom 10px;
+  height: 200px;
+  background-size: 100px 200px;
+  background-size: 200px 100px;
+}
+
+.langs h6 {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  margin: 0;
+}
+
+.langs {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.langs svg {
+  margin: 0 10px;
+}
+
+.contact {
+  padding: 18px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.contact button {
+  padding: 12px;
+  background: #ff6b00;
+  margin: 7px 0;
+  border: 0 solid transparent;
+  color: white;
+}
+
+/*
+.contact {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+} */
+
+.form-info {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.form-info input {
+  margin: 10px;
+  border-width: 0 0 2px 0;
+  border-style: solid;
+  border-color: lightgray;
+  height: 30px;
+  width: 100%;
+}
+
+.form-info textarea {
+  margin: 10px;
+  border-width: 0 0 2px 0;
+  border-style: solid;
+  border-color: lightgray;
+  height: 70px;
+  width: 100%;
+}
+
+.form {
+  width: 80%;
+  display: inline-block;
+}
+
+.framwork-links span {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  color: #091e42;
+  border: 1px solid lightgray;
+  padding: 10px;
+}
+
+.reads {
+  padding: 28px 18px;
+  color: white;
+  width: 70%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 10px;
+}
+
+.framwork-links {
+  margin: 10px 0;
+}
+
+.ul-links {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  justify-content: flex-start;
+  margin: 0;
+  padding: 0;
+}
+
+.ul-links li {
+  margin: 1px;
+  background-color: lightgray;
+  padding: 5px;
+}
+
+.btn-see {
+  border: 0;
+  width: 100%;
+  color: white;
+  height: 30px;
+  background: #ff6b00;
+  cursor: pointer;
+}
+
+.card-details {
+  margin: 0 15px;
+}
+
+.card {
+  display: inline-block;
+  margin: 10px 0;
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%);
+}
```

Comment:
- [ ] Kindly adjust the with of the cards to be similar to the Figma template.

> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

- Your design
![image](https://user-images.githubusercontent.com/58642949/187033872-81e76c8d-b5ed-4354-b2bf-28d683da4f14.png)